src/variorum/Nvidia/Volta.h

```diff
@@ -0,0 +1,10 @@
+#ifndef VOLTA_H_INCLUDE
+#define VOLTA_H_INCLUDE
+
+int volta_get_power(int long_ver);
+int volta_get_thermals(int long_ver);
+int volta_get_clocks(int long_ver);
+int volta_get_power_limits(int long_ver);
+int volta_get_gpu_utilization(int long_ver);
+
+#endif
```

Comment:
Add extra space between each function declaration in header files:
```
int volta_get_power(int long_ver);

int volta_get_thermals(int long_ver);

int volta_get_clocks(int long_ver);

int volta_get_power_limits(int long_ver);

int volta_get_gpu_utilization(int long_ver);
```

Reply:
Fixed.